TAO/TAO_IDL/be/be_visitor_structure/cdr_op_cs.cpp

```diff
@@ -81,7 +83,7 @@ be_visitor_structure_cdr_op_cs::visit_structure (be_structure *node)
                         -1);
     }
 
-  *os << ";" << be_uidt << be_uidt_nl
+  *os << "true;" << be_uidt << be_uidt_nl
```

Comment:
Why a true here?

Reply:
It's required for how the visitor creates the expressions. This is the case for the empty struct.